breaddit/views.py

```diff
@@ -0,0 +1,69 @@
+from django.db import connection
+from django.http import JsonResponse
+
+
+def dictfetchall(cursor):
+    """
+    Return all rows from a cursor as a dict.
+    Assume the column names are unique.
+    """
+    columns = [col[0] for col in cursor.description]
+    return [dict(zip(columns, row)) for row in cursor.fetchall()]
+
+
+def comments(request, post_id):
+    """
+    Get all comments for a post, along with the user who created the comment.
+
+    :param request: HttpRequest
+    :param post_id:
+    :return:
+    """
+
+    post_comment_sql = (
+        "SELECT id, body, created_at, updated_at, author_id "
+        "FROM breaddit_comment "
+        "WHERE post_id = {} and status <> 'deleted'"
+    ).format(post_id)
+
+    with connection.cursor() as cursor:
+        cursor.execute(post_comment_sql)
+        comments = dictfetchall(cursor)
+
+    users = []
+
+    for comment in comments:
+        user_sql = (
```

Comment:
same as above:
Definitely want to use ORM here instead of raw SQL to avoid SQL injection possibility.

Reply:
Again , would not need to query for comments as well if we either join or use ORM ( which just joins).